UI.NUnitVersion/Video/EndToEndTest.cs

```diff
@@ -63,7 +117,21 @@ private void BookHearing(BookingDto bookingDto)
         var summaryPage = createHearingPage.EnterHearingDetails(bookingDto);
         var confirmationPage = summaryPage.ClickBookButton();
         _hearingIdString = confirmationPage.GetNewHearingId();
-        confirmationPage.ClickViewBookingLink();
+        var bookingDetailsPage = confirmationPage.ClickViewBookingLink();
+        bookingDetailsPage.GoToDashboardPage();
+        
+        var teamMemberUsername = WorkAllocationTestData.JusticeUserUsername;
+        
+        _justiceUser = await CreateVhTeamLeaderJusticeUserIfNotExist(teamMemberUsername);
+        _conference = await VideoApiClient.GetConferenceByHearingRefIdAsync(new Guid(_hearingIdString) , false);
+ 
+        var manageWorkAllocationPage = dashboardPage.GoToManageWorkAllocation();
+        
+        manageWorkAllocationPage.AllocateJusticeUserToHearing(
+            caseNumber: _conference.CaseNumber,
+            justiceUserDisplayName: _justiceUser.FullName,
+            justiceUserUsername: _justiceUser.Username);
+        AdminWebDriver.Terminate();
```

Comment:
Don't terminate in the test, the base class handles that in one go. Also I think if you terminate early you won't be able to invoke the JS to report test results. See the TearDown method in `VideoWebUiTest`